bin/install-macos.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+echo 'installing brew packages'
+brew install tfenv tflint terraform-docs pre-commit
+
+echo 'installing pre-commit hooks'
+pre-commit install
+
+echo 'installing terraform with tfenv'
+tfenv install
```

Comment:
This is copypasta from before. Here's the new one https://github.com/rhythmictech/terraform-anycloud-template/blob/master/bin/install-macos.sh

Reply:
This is also why the build is failing 
```
Terraform validate with tfsec............................................Failed
- hook id: terraform_tfsec
- exit code: 127

/Users/runner/.cache/pre-commit/repojehn6r5j/terraform_tfsec.sh: line 32: tfsec: command not found
/Users/runner/.cache/pre-commit/repojehn6r5j/terraform_tfsec.sh: line 32: tfsec: command not found
/Users/runner/.cache/pre-commit/repojehn6r5j/terraform_tfsec.sh: line 32: tfsec: command not found
/Users/runner/.cache/pre-commit/repojehn6r5j/terraform_tfsec.sh: line 32: tfsec: command not found
```